packages/open-truss/src/uqi/uqi.ts

```diff
@@ -0,0 +1,176 @@
+import Iterator from './iterator'
+
+export type UqiMappedType =
```

Comment:
I forget, we decided to not support ArrayBuffers correct? Maybe worth adding a brief comment here saying that is intentional?

That being said, I feel like we'll have to support that sooner than later to process files. But that seems like a problem to tackle when we get there.

Reply:
I'd like to support them sooner rather than later but this PR was already a beast so thought it could be an addition in the near future once I got to a data source that required it.